packages/jest-runtime/src/transform.js

```diff
@@ -159,6 +161,18 @@ const shouldInstrument = (filename: Path, config: Config): boolean => {
   }
 
   if (
+    !config.collectCoverageOnlyFrom && // still cover if `only` is specified
+    config.collectCoverageFrom &&
+    !multimatch(
```

Comment:
How's the performance of multimatch compared to regex?

Would it be faster to convert a glob pattern to a regex once and then apply it to file names later?